### DISCUSSION TITLE
Where to put link to DAO Documents?

### DISCUSSION THREAD

**Discussion by plor** [odyssy-automaton/pokemol-web#1859] [General]
The Poster boost currently requires the user to go into boost settings to view documents. The UX review suggested adding a link on the navbar (left side) when this boost is activated. There was disagreement about adding more to the navbar, but where to put it was never finalized. If you have opinions or ideas about where best to place this, please add them here and we can arrive to agreement.

**Comment by earth2travis** [comment]
Seems like an important section of the DAO that would warrant easy access. I do not see a problem with adding it to the left navigation for DAOs that have the boost installed. Ultimately, we need to provide access to the documents in context. A manifesto on the Overview. Possibly a description of the proposal process on the Proposals view. A description of the treasury and minions on the Vaults view. A "How to Join" on the Member view. Et cetera. But it would be a good idea to provide a link to all the Docs in the navigation.

**Reply by plor** [reply]
Thanks for the response. I agree that we can build out a robust set of "document types" that can be presented in different locations. The main location of all documents seems like it could live on the navbar.

**Reply by skuhlmann** [reply]
I like the idea of the side nav as well if we only show it for daos that have the boost active.
